docs/azure/sdk/resource-management.md

```diff
@@ -153,7 +179,7 @@ await foreach(VirtualMachine vm in vmCollection.GetAllAsync())
 }
  ```
 
-#### Structured Resource Identifier
+### Structured Resource Identifier
 
 Resource IDs contain useful information about the resource itself, but they're plain strings that have to be parsed. Instead of implementing your own parsing logic, you can use a `ResourceIdentifier` object that will do the parsing for you: `new ResourceIdentifier("myid");`.
```

Comment:
```suggestion
Resource IDs contain useful information about the resource itself, but they're plain strings that have to be parsed. Instead of implementing your own parsing logic, you can use a `ResourceIdentifier` object that will do the parsing for you.
```
A detailed sample is provided below, I don't think this is needed here anymore.